src/LazyLoadingCommand.php

```diff
@@ -55,4 +60,29 @@ protected function execute(InputInterface $input, OutputInterface $output) : int
 
         return $command->execute($input, $output);
     }
+
+    public function getCommandClass() : string
+    {
+        return $this->commandClass;
+    }
+
+    /**
+     * @throws RuntimeException
+     */
+    public function __invoke(ConsoleEvent $event) : int
+    {
+        if (! is_a($this->commandClass, CommandListenerInterface::class, true)) {
```

Comment:
instanceof seems faster than `is_a`, see https://3v4l.org/Niuec . Also, for consistency with other components, the instanceof is mostly used instead.

Reply:
`instaceof` does not work on strings, we need to have an object. Here, at this point, we just have a class name (string), not an object yet.

Reply:
ok 👍